Sources/TuistGenerator/Generator/LinkGenerator.swift

```diff
@@ -245,21 +245,29 @@ final class LinkGenerator: LinkGenerating { // swiftlint:disable:this type_body_
         }
 
         if !frameworkReferences.isEmpty {
-            let precompiledEmbedPhase = PBXShellScriptBuildPhase(name: "Embed Precompiled Frameworks")
+            for reference in frameworkReferences {
+                guard case let GraphDependencyReference.framework(path, _, _, _, _, _, _, _, _) = reference
```

Comment:
this check is actualy taked from the `embedScriptGenerator`. Inside it there's the same check